robocorp-code/vscode-client/src/inspector/inspectorView.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Interesting docs related to webviews:
+ * https://code.visualstudio.com/api/extension-guides/webview
+ */
+import path = require("path");
+import { readFileSync } from "fs";
+
+import * as vscode from "vscode";
+
+import { getExtensionRelativeFile, verifyFileExists } from "../files";
+import { logError } from "../channel";
+import { getSelectedRobot } from "../viewsCommon";
+import { LocatorsMap } from "./types";
+import { IMessage, IMessageType } from "./protocols";
+
+export async function showInspectorUI(context: vscode.ExtensionContext) {
+    const panel = vscode.window.createWebviewPanel(
+        "robocorpCodeInspector",
+        "Robocorp Inspector",
+        vscode.ViewColumn.One,
+        {
+            enableScripts: true,
+            retainContextWhenHidden: true,
+        }
+    );
+
+    const robot = getSelectedRobot();
+    if (robot) {
+        let locatorJson = path.join(robot.robot.directory, "locators.json");
+        if (verifyFileExists(locatorJson, false)) {
+            vscode.workspace.openTextDocument(vscode.Uri.file(locatorJson)).then((document) => {
+                let text = document.getText();
+                panel.webview.html = getWebviewContent(JSON.parse(text) as LocatorsMap);
+            });
+        } else {
+            logError("locators.json.not.found", undefined, "");
```

Comment:
I guess that this should not be considered an error... If the `locators.json` is not there it shouldn't be a problem as it should be created right?

p.s.: The `logError` is something that'll end up in our errors sent from RCC, which is not what we want (the `errorCode` in this case should be something significant to find the source afterwards, not an empty string).

p.s.: If you just want to do a regular log, use `OUTPUT_CHANNEL.appendLine` -- this will make the message appear in `Views > OUTPUT > Robocorp Code`.